doc/method_documentation.rdoc

```diff
@@ -181,3 +181,38 @@ of the method.
 Methods are documented using RDoc syntax.  See the
 {RDoc Markup Reference}[https://docs.ruby-lang.org/en/master/RDoc/Markup.html#class-RDoc::Markup-label-RDoc+Markup+Reference]
 for more information on formatting with RDoc syntax.
+
+=== Output from irb
+
+Consider whether <tt># => ...</tt> in successive codeblock lines should be aligned.
+Alignment may sometimes aid readability.
+
+=== Lists
+
+A list should be preceded by and followed by a blank line.
+This is unnecessary for the HTML output, but helps in the +ri+ output.
+
+=== Auto-Links
+
+Consider whether an auto-link (e.g., to +Array+)
+should be suppressed (<tt>\Array</tt>) or allowed (Array).
+
+It may be best to suppress when local;
+that is, in the documentation for class \Array, suppress auto-links to \Array.
+
+For an "off-class" reference, the decision to suppress or not may depend on context.
+
+For some classes, this decision may be avoided
+by using an \English word instead of a class name
+(e.g., _array_ instead of _Array_, _string_ instead of _String_).
+
+=== Call-Seq
+
+A +call-seq+ block should have <tt>{|x| ... }</tt>, not <tt>{|x| block }</tt> or <tt>{|x| code }</tt>.
+
+A +call-seq+ output should:
+- Have +self+, not +receiver+ or +array+.
+- Begin with +new_+ if the output object is a new instance of the receiver's class,
+  to emphasize that the output object is not +self+.
+
+A +call-seq+ output may begin with +new_+ for any object that actually is new.
```

Comment:
If we are going to discuss this, we should probably say it should only start with `new_` if the object is a new instance of the receiver's class.  For example, `Hash#to_a` should not use `new_array`, it should use `array`.

Reply:
Ok with me.  but I have a year's worth of the opposite.

Reply:
Fixed.